lesson06/script.js

```diff
@@ -0,0 +1,25 @@
+const lists = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+];
+
+const promise = new Promise(resolve => {
+    setTimeout(() => resolve(lists),3000);
+});
```

Comment:
```suggestion
const promise = new Promise(resolve => {
    const lists = [
    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
];

    setTimeout(() => resolve(lists),3000);
});
```
Nits
I can summarize it.

Reply:
Thank you for your comment!
Put the required code in Promise and
Very easy to read!
I will try to summarize.